libs/dh/metering-point/feature-overview/src/lib/secondary-master-data/dh-secondary-master-data.component.html

```diff
@@ -193,12 +165,8 @@
           {{ t('ratedCapacity') }}
         </dt>
         <dd class="watt-text-m">
-          <ng-container
-            *ngIf="secondaryMasterData?.ratedCapacity; else emptyField"
-          >
-            {{ secondaryMasterData?.ratedCapacity }}
-            {{ t('units.kilowatt') }}
-          </ng-container>
+          {{ secondaryMasterData?.ratedCapacity | emDashFallback }}
```

Comment:
Question: wouldn't this change cause kilowatt to be shown even though ratedCapacity is not defined?

Reply:
Yes. good catch. I think the options are similar to my previous comment above (under `capacity`).